### PR TITLE
[Jetsnack] Add Previews

### PR DESCRIPTION
**Pull request by JolandaVerhoef**

### CODE REVIEW

**Review by nickbutcher — APPROVED** [review]
One question about API level preview, otherwise LGTM.

**Comment by JolandaVerhoef** [comment]
For me, it made sense to include the minSdkVersion vs the targetSdkVersion. They shouldn't differ, but if you forget to use Compat classes, they might. I don't have a strong feeling about this though. Would like to get @yrezgui 2 cents on this!

File: Jetsnack/app/src/main/java/com/example/jetsnack/ui/components/Button.kt [hunk 1]
```diff
@@ -99,3 +103,29 @@ fun JetsnackButton(
 }
 
 private val ButtonShape = RoundedCornerShape(percent = 50)
+
+@Preview("default", "round")
+@Preview("dark theme", "round", uiMode = UI_MODE_NIGHT_YES)
+@Preview("api 21", "round", apiLevel = 21)
```
[the remark this answers is not in the document]
Comment (yrezgui) [reply Jetsnack/app/src/main/java/com/example/jetsnack/ui/components/Button.kt:109]: I think we don't have an actual need in this sample as we don't rely on changing APIs within our previews. Better to drop it

**Review by yrezgui — CHANGES_REQUESTED** [review]
I would remove `@Preview("api 21", "round", apiLevel = 21)` as we don't have an actual need in this sample as we don't rely on changing APIs within our previews. Better to drop them

**Comment by JolandaVerhoef** [comment]
@yrezgui I removed the API level previews. Let me know if this is better!

**Review by yrezgui — APPROVED** [review]

**Pull request merged**